lib/pwwka/tasks.rb

```diff
@@ -6,6 +6,8 @@
     handler_klass = ENV['HANDLER_KLASS'].constantize
     queue_name    = "#{ENV['QUEUE_NAME']}_#{Rails.env}"
     routing_key   = ENV['ROUTING_KEY'] || "#.#"
+    Pwwka.configuration.prefetch = ENV['PREFETCH'] if ENV.has_key?('PREFETCH')
```

Comment:
If this is a setting per handler, should it be a param to the `subscribe` method instead of a global configuration value?  Perhaps with global configuration setting a default?

Reply:
That seems reasonable to me.

Reply:
Cool.  The default should be `nil` for backwards-compat.  We can push out app-wide configuration changes via the messaging gem (I'm assuming there is a better system-wide default than `nil`)